src/condor_tools/htcondor_cli/annex_create.py

```diff
@@ -0,0 +1,720 @@
+#!/usr/bin/env python3
+
+
+import os
+import sys
+import fcntl
+import atexit
+import signal
+import getpass
+import logging
+import secrets
+import subprocess
+from pathlib import Path
+
+import htcondor
+
+
+INITIAL_CONNECTION_TIMEOUT = int(
+    htcondor.param.get("ANNEX_INITIAL_CONNECTION_TIMEOUT", 180)
+)
+REMOTE_CLEANUP_TIMEOUT = int(htcondor.param.get("ANNEX_REMOTE_CLEANUP_TIMEOUT", 60))
+REMOTE_MKDIR_TIMEOUT = int(htcondor.param.get("ANNEX_REMOTE_MKDIR_TIMEOUT", 5))
+REMOTE_POPULATE_TIMEOUT = int(htcondor.param.get("ANNEX_REMOTE_POPULATE_TIMEOUT", 60))
+
+# FIXME: Make this come from config? At least some place not hardcoded.
+MACHINE_QUEUE_MAP = {"stampede2": "development"}
```

Comment:
```suggestion
MACHINE_QUEUE_MAP = {"stampede2": "normal"}
```
The `development` queue turns around faster but only allows one job.